pkg/ts/utils/object.ts

```diff
@@ -78,7 +78,7 @@ export function deepCopy<T>(datum: T): T {
   } else {
     return datum;
   }
-};
+}
```

Comment:
Whoops, bad Rust habit. Thanks for fixing that

Reply:
NBD, I just updated the lint to highlight semis.